docs/pages/versions/unversioned/sdk/ui.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: UI
+description: A set of platform components to build native-feeling UIs.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-ui'
+packageName: '@expo/ui'
+platforms: ['android', 'ios']
+---
+
+import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+
+`@expo/ui` is a set of native input components that allows you to build native-feeling forms.
```

Comment:
In the page's description we say native feeling "UIs" and here we are referring to "forms". I think it might lead to confusion so let's update here or the description to match the verbiage.